db/corruption_test.cc

```diff
@@ -824,6 +831,43 @@ TEST_F(CorruptionTest, DisableKeyOrderCheck) {
   ROCKSDB_NAMESPACE::SyncPoint::GetInstance()->ClearAllCallBacks();
 }
 
+TEST_F(CorruptionTest, VerifyWholeTableChecksum) {
+  delete db_;
+  db_ = nullptr;
+  Options options;
```

Comment:
Please either use Options options = GetDefaultOptions() or be sure to set env_.  Otherwise, this test will fail for the non-default Env cases

Reply:
Good catch!